app/controllers/hyrax/downloads_controller.rb

```diff
@@ -41,6 +41,7 @@ def derivative_download_options
       # Hydra::Ability#download_permissions can't be used in this case because it assumes
       # that files are in a LDP basic container, and thus, included in the asset's uri.
       def authorize_download!
+        return if current_ability.admin?
```

Comment:
Can this be moved into the Ability class? It seems like the short-circuit should be there.